app.py

```diff
@@ -1,29 +1,53 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, render_template, request, redirect
 import session_items as session
+import requests
+import trello_utils
+from trello_utils import Card, getCardStatus
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
More subtly:

You could also delete this line, the flask_config.py file and any mention of `SECRET_KEY` in the README / .env.template
SECRET_KEY is used by Flask to encrypt the cookie that we were previously using to store the to do list. We're no longer using it so we don't need that.